build.sh

```diff
@@ -29,6 +29,7 @@ HELP="$0 [<target> ...] [<flag> ...] [--cmake-args=\"<args>\"] [--cache-tool=<to
    docs             - build the documentation
    tests            - build the tests
    bench            - build the benchmarks
+   uninstall        - uninstall any files from a previous installation
```

Comment:
This doesn't seem correct as we only uninstall the libraft components and not any python components ( pylibraft / raft-dask ).